demo/scripts/controls/sidePane/editorOptions/EditorOptionsPlugin.ts

```diff
@@ -1,64 +1,64 @@
-import BuildInPluginState, { BuildInPluginProps, UrlPlaceholder } from '../../BuildInPluginState';
-import getDefaultContentEditFeatureSettings from './getDefaultContentEditFeatureSettings';
-import OptionsPane from './OptionsPane';
-import SidePanePluginImpl from '../SidePanePluginImpl';
-import { ExperimentalFeatures } from 'roosterjs-editor-types';
-import { SidePaneElementProps } from '../SidePaneElement';
-
-const initialState: BuildInPluginState = {
-    pluginList: {
-        contentEdit: true,
-        hyperlink: true,
-        paste: true,
-        watermark: false,
-        imageEdit: true,
-        cutPasteListChain: true,
-        tableCellSelection: true,
-        tableResize: true,
-        customReplace: true,
-        pickerPlugin: true,
-        listEditMenu: true,
-        imageEditMenu: true,
-        tableEditMenu: true,
-        contextMenu: true,
-        autoFormat: true,
-    },
-    contentEditFeatures: getDefaultContentEditFeatureSettings(),
-    defaultFormat: {},
-    linkTitle: 'Ctrl+Click to follow the link:' + UrlPlaceholder,
-    watermarkText: 'Type content here ...',
-    forcePreserveRatio: false,
-    experimentalFeatures: [
-        ExperimentalFeatures.PasteWithLinkPreview,
-        ExperimentalFeatures.SingleDirectionResize,
-        ExperimentalFeatures.ImageRotate,
-        ExperimentalFeatures.ImageCrop,
-        ExperimentalFeatures.ConvertSingleImageBody,
-        ExperimentalFeatures.TableAlignment,
-        ExperimentalFeatures.AdaptiveHandlesResizer,
-        ExperimentalFeatures.ListItemAlignment,
-    ],
-    isRtl: false,
-};
-
-export default class EditorOptionsPlugin extends SidePanePluginImpl<
-    OptionsPane,
-    BuildInPluginProps
-> {
-    constructor() {
-        super(OptionsPane, 'options', 'Editor Options');
-    }
-
-    getBuildInPluginState(): BuildInPluginState {
-        let result: BuildInPluginState;
-        this.getComponent(component => (result = component.getState()));
-        return result || initialState;
-    }
-
-    getComponentProps(base: SidePaneElementProps) {
-        return {
-            ...initialState,
-            ...base,
-        };
-    }
-}
+import BuildInPluginState, { BuildInPluginProps, UrlPlaceholder } from '../../BuildInPluginState';

```

Comment:
Please check why the whole file is shown as changed, this makes it hard to see what is the real change

Reply:
I do not know how to fix this, I only changed the string 'Ctrl+Click to follow the link:' to 'Ctrl+Click to follow the link: '.

Reply:
I think this usually happens if you have autocrlf disabled. Try using this config: 
git config --global core.autocrlf true